open-metadata-test/open-metadata-fvt/access-services-fvt/discovery-engine-fvt/build.gradle

```diff
@@ -0,0 +1,109 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * Copyright Contributors to the ODPi Egeria project.
+ */
+
+plugins {
+    // Include plug-in for starting a new process
+    id 'com.github.hesch.execfork'
```

Comment:
the name of the plugin has changed - and further details in the fvt execution itself. see https://github.com/odpi/egeria/commit/4834f05815ac439b5dd3b49ddb442dd8c9ccbc10